### DISCUSSION TITLE
[FEATURE REQ] Indicate Errors in UI - I.e. Feeds Failing

### DISCUSSION THREAD

**Discussion by bakerboy448** [autobrr/autobrr#1659] [Ideas]
**Is your feature request related to a problem? Please describe.**
![image](https://user-images.githubusercontent.com/55419169/225080062-4149d225-7a4b-421a-a7a0-45591b8fcf26.png)
No indication in the UI for why newznab was not working

**Describe the solution you'd like**
UI should indicate an error exists and the details

**Describe alternatives you've considered**
Having to search logs

**Additional context**
relevant log entry for why this was not working and had no indication in the ui

```
{"level":"error","module":"feed","feed":"UseNetIndexer (Prowlarr)","error":"could not get feed: could not make request. &{Method:GET URL:http://127.0.0.1/prowlarr/11/api?apikey=KEY&t=search Proto:HTTP/1.1 ProtoMajor:1 ProtoMinor:1 Header:map[] Body:<nil> GetBody:<nil> ContentLength:0 TransferEncoding:[] Close:false Host:127.0.0.1 Form:map[] PostForm:map[] MultipartForm:<nil> Trailer:map[] RemoteAddr: RequestURI: TLS:<nil> Cancel:<nil> Response:<nil> ctx:0xc000126000}: Get \"https://127.0.0.1/prowlarr/11/api?apikey=KEY&t=search\": tls: failed to verify certificate: x509: cannot validate certificate for 127.0.0.1 because it doesn't contain any IP SANs","time":"2023-03-14T08:00:25-05:00","message":"error fetching feed items"}
```

**Comment by ghost** [comment]
This shouldn't get placed in the Last Run column. This needs to be a separate column, preferably preceding Enabled.
